src/modules/commander/Commander.cpp

```diff
@@ -150,7 +150,7 @@ static bool send_vehicle_command(const uint32_t cmd, const float param1 = NAN, c
 	vcmd.source_system = vehicle_status_sub.get().system_id;
 	vcmd.target_system = vehicle_status_sub.get().system_id;
 	vcmd.source_component = vehicle_status_sub.get().component_id;
-	vcmd.target_component = vehicle_status_sub.get().component_id;
+	vcmd.target_component = 161; // MAV_COMP_ID_PARACHUTE
```

Comment:
We can't hard code this here right? We should best refer to the actual parachute ID from the MAVLink spec.

Reply:
We don't have the MAVLink headers here, so we would have to duplicate the constant somewhere in a uORB topic, or just hard-code it like this.

Reply:
As just discussed in person this breaks other places `send_vehicle_command` is used because it hardcodes the parachute component ID.

Reply:
Fixed in cd7c74ffb4891e5ed249bbdaa6155eb13d81d553.